src/winit_input_helper.rs

```diff
@@ -89,13 +99,16 @@ impl WinitInputHelper {
         for event in events {
             self.process_window_event(event);
         }
+        self.end_step();
     }
 
     fn step(&mut self) {
         self.dropped_file = None;
         self.window_resized = None;
         self.scale_factor_changed = None;
         self.close_requested = false;
+        self.step_start.get_or_insert(Instant::now());
```

Comment:
Can this be `self.step_start = Some(Instant::now())`? Using get_or_insert for this is confusing.

Reply:
Oh wait I misunderstood

Reply:
This sets step_start to some when it's none to ensure that the first tick has a duration?
Can we just set it in the constructor instead? Or are we trying that avoid the first tick appearing excessively long?

Reply:
Yeah that's what I was trying to avoid

Reply:
I think you are making the right call here. Can you add a comment above this line something like "set step_some on the first event in attempt to give a more realistic length for the first tick"